CalculadoraCarbono/miapp/fix_serializers.py

```
@@ -0,0 +1,85 @@
+            'reduccion_por_reciclaje', 'detalle_consumo', 'detalle_transporte',
+            'detalle_energia', 'detalle_residuos', 'detalle_consumo_input',
+            'detalle_transporte_input', 'detalle_energia_input', 'detalle_residuos_input'
+        ]
+        read_only_fields = (
+            'id', 'usuario', 'huella_total', 'huella_consumo', 'huella_transporte',
+            'huella_energia', 'huella_residuos', 'reduccion_por_reciclaje', 'fecha'
+        )
+    
+    def get_detalle_consumo(self, obj):
+        try:
+            return DetalleConsumoSerializer(obj.detalle_consumo).data
```

Ruff (0.8.2)

Undefined name `DetalleConsumoSerializer`

(F821)